src/graphics/bargraphics.jl

```diff
@@ -1,3 +1,5 @@
+if VERSION < v"0.6-"
```

Comment:
what would be a way to find a reasonable lower bound? any pointer where to start looking. Is it related to the merge of https://github.com/JuliaLang/julia/pull/18457 ?